app/components/task-assignment.js

```diff
@@ -67,6 +71,18 @@ export default Component.extend({
       }
     },
 
+    closeDropdown() {
+      this.sendAction('onclose');
+      let that = this;
+      run.next(that, function() {
```

Comment:
could this work as `next(that, () =>` and keep the scope so you don't have to have an intermediate variable here?